modules/imaging_browser/php/imagingbrowserrowprovisioner.class.inc

```diff
@@ -36,8 +36,13 @@ class ImagingBrowserRowProvisioner extends \LORIS\Data\Provisioners\DBRowProvisi
      */
     function __construct()
     {
-        $config = \NDB_Config::singleton();
-
+        $config   = \NDB_Config::singleton();
+        $instance = null;
+        try {
+            $instance = \NDB_Factory::singleton()->database();
```

Comment:
$instance is very non-specific here. your better off using `$DB` or `$db` instead